src/styles/block/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 400px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__top {
+    display: flex;
+    justify-content: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__top-img {
+    width: 100%;
+  }
+
+  &__bottom {
+    margin-top: 40px;
+
+    &__tittle {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $mainAccent;
+    }
+
+    &__bottom-code {
+      margin-top: 4px;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $secondary;
+    }
+
+    &-stars {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      margin-top: 16px;
+    }
+
+    &-text {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $mainAccent;
+    }
+
+    &-cost {
+      display: flex;
+      margin-top: 24px;
+      justify-content: space-between;
+    }
+
+    &-price {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary;
+    }
+
+    &-sum {
+      font-size: 16px;
+      font-weight: 800;
+      line-height: 18px;
+    }
+
+    &-button {
```

Comment:
The class name `&-button` should be corrected to `&__bottom-button` to match the HTML structure `card__bottom-button`.